metadata-load/src/test/java/uk/ac/ebi/ampt2d/metadata/importer/api/SraObjectsImporterThroughAPITest.java

```diff
@@ -94,4 +99,16 @@ public void importAnalysisObject() throws Exception {
         assertEquals(0, studyRepository.count());
         assertEquals(0, analysisRepository.count());
     }
-}
\ No newline at end of file
+
+    @Test
+    public void importReferenceSequenceObject() throws Exception {
```

Comment:
also, add asserts in importStudy to check whether reference sequences are imported?

Reply:
this is not done yet

Reply:
Actually, currently no reference sequences are imported in `importStudy` , because two studies don't have any analyses (SRP000392, SRP000118), and the remaining study ERP000054 has only one analysis ERZ000001 which has a custom (currently non-importable) reference sequence:
```
               <ASSEMBLY>
                    <CUSTOM>
                         <URL_LINK>
                              <LABEL>Mus_musculus.NCBIM37.52</LABEL>
                              <URL>ftp://ftp.ensembl.org/pub/release-52/fasta/mus_musculus/dna/Mus_musculus.NCBIM37.52.dna.toplevel.fa.gz</URL>
                         </URL_LINK>
                    </CUSTOM>
               </ASSEMBLY>
```

To add this test, I replaced `ERP000054` with `ERP006576`, which has two analyses and one reference sequence.